.env

```diff
@@ -0,0 +1,4 @@
+# Disable React Host Check in Dev Mode
```

Comment:
I don't remember adding this and don't have it locally? not sure how it ended up here.